### ISSUE TITLE
DEV | Linear Regression

### ISSUE THREAD

**Issue by AlexisTercero55** [AlexisTercero55/AI-Research#2]
# Requirements for Linear regresion model 

- [ ] Normal equation implementation.
- [ ] Gradient based learning.
- [ ] At least two module implementation.
- [ ] Math documentation.

**Comment by AlexisTercero55** [comment]
0a79b37eeb0c57c1c67ed4a3eef750aeeaac61c0

**Comment by AlexisTercero55** [comment]
## At least two module implementations

- [ ] sklearn.linear_model
- [ ] TF | Torch
